packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1887,24 +1663,6 @@ export class ProjectService extends BaseService {
                         );
                     }
 
-                    /**
-                     * If the feature flag is enabled, and we actually have any table calculations
-                     * to process, we use the new in-memory table calculations engine. This avoid us
-                     * spinning-up a new DuckDB database pointlessly.
-                     *
-                     * In a follow-up after initial testing, this check should be done somewhere further
-                     * down the stack.
-                     */
-                    const newTableCalculationsFeatureFlagEnabled =
-                        await isFeatureFlagEnabled(
-                            FeatureFlags.UseInMemoryTableCalculations,
```

Comment:
We can remove the feature flag `UseInMemoryTableCalculations` from `FeatureFlags`